Lib/corpuscrawler/crawl_gd.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 Google Inc. All rights reserved.
+# Copyright 2017 Jim O'Regan
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, print_function, unicode_literals
+import re
+
+from corpuscrawler.util import crawl_bbc_news, crawl_udhr
+
+
+def crawl(crawler):
+    out = crawler.get_output(language='gd')
+    # UDHR not on the Unicode site yet, but is translated:
+    # http://www.ohchr.org/EN/UDHR/Pages/Language.aspx?LangID=gls
+    #crawl_udhr(crawler, out, filename='udhr_gls.txt')
```

Comment:
Looks like the UDHR is available under `gla` tag:
https://unicode.org/udhr/d/udhr_gla.html